doc/user/content/demos/business-intelligence.md

```diff
@@ -136,17 +136,19 @@ Putting this all together, our deployment looks like this:
    ```
 
    Note that downloading the Docker images necessary for the demo can take quite
-   a bit of time (upwards of 10 minutes, even on very fast connections).
+   a bit of time (upwards of 10 minutes, even on fast connections).
+
+   When the script exits successfully, all components from the diagram above are
+   active in Docker containers, and the chBench client is rapidly pushing data 
+   and transactions to the MySQL database.
 
 ### Define sources & views
 
 Now that our deployment is running (and looks like the diagram shown above), we
-can get Materialize to read data from Kafka, and define the views we want
+can get Materialize to read data from Kafka and define the views we want
 Materialize to maintain for us.
 
-1. Launch a new terminal window and `cd <path to materialize>/demo/chbench`.
-
-1. Launch the Materialize CLI (`mzcli`) by running:
+1. In the same terminal, launch the Materialize CLI (`mzcli`) by running:
```

Comment:
Since you’re in here, I think this recently changed to use psql instead of mzcli. Maybe drop the parenthetical so this just says “Materialize CLI” and won’t go stale in the future?